docs/migrations.md

```diff
@@ -41,7 +41,7 @@ gradle build -i
 An example command to run the jar against the `imrt` database: 
 
 ```
- java -jar <location of jar>/AP_IMRT_Schema.jar --spring.datasource.url="jdbc:postgresql://localhost:5432/imrt" --spring.datasource.username="<user>" --spring.datasource.password="<password>"
+ java -jar <location of jar>/AP_IMRT_Schema.jar --spring.datasource.url="jdbc:postgresql://localhost:5432/imrt" --spring.datasource.username="imrt_admin" --spring.datasource.password="<password>"
```

Comment:
I'm going to merge this change since this is how Fairway does local dev.  However, any user can be used.  It is just that our getting started notes recommend using imrt_admin.  You could create "bogus_user" as long as that bogus user has the proper privelages this all still works.  Nothing in this project is hardcoded to use imrt_admin.